src/screens/profile/components/user-info/user-info.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Avatar } from 'react-native-paper';
+
+import { Heading, Text } from 'src/components';
+import { AppColor, HeadingLevel } from 'src/common/enums';
+
+import styles from './styles';
+
+const UserInfo = () => {
+  return (
+    <View style={styles.container}>
+      <Avatar.Image
+        size={148}
+        source={{
+          uri: 'https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png',
+        }}
```

Comment:
Replace it with [user_profile](https://fonts.google.com/icons?selected=Material+Icons&icon.query=account_circle) icon.